Slash/Utility/Comments/Comments.pm

```diff
@@ -2656,16 +2202,6 @@ sub isTroll {
 }
 
 
-########################################################
```

Comment:
This particular sub, once in a metamod template, which is fine but I'll remove it anyway. It is called a couple places I missed in Comments.pm now that I look though and those bits of logic need to go away as well.